### ISSUE TITLE
Issues, wrong parameters, files being in used etc.. Ran as admin and Trusted installer, logs inside.

### ISSUE THREAD

**Issue by wazerstar** [ntdevlabs/tiny11builder#162]
I decided to test the script using the latest uup  Windows 11, version 23H2 (22631.3527) amd64 Danish version

This is a clean and fresh windows 11 Danish Pro version btw.

Ran powershell as admin

Did

`set-executionpolicy remotesigned` and picked All

`Set-ExecutionPolicy` unrestricted and picked All

The log from just running powershell as admin with following commands above..

[normal_admin_tiny11.log](https://github.com/ntdevlabs/tiny11builder/files/15181711/normal_admin_tiny11.log)

I decided to test with [nsudo](https://github.com/M2TeamArchived/NSudo) with TrustedInstaller Rights running powershell at highest possible level.

It did a bit better but still bugs.

[nsudo_trustedinstaller_admin_tiny11.log](https://github.com/ntdevlabs/tiny11builder/files/15181721/nsudo_trustedinstaller_admin_tiny11.log)

I'm not going to bother installing this because it still says wrong parameters

One of the issues I noticed is that you would need to call an universal cmd for Administrators since In Danish its called Administratorer.
I guess you need to parse/list what its called into an array or something and then pick what its called.

I also found issues with

```
Loading registry...
ERROR: The system was unable to find the specified registry key or value.
ERROR: The system was unable to find the specified registry key or value.
ERROR: The system was unable to find the specified registry key or value.
ERROR: The system was unable to find the specified registry key or value.
ERROR: The system was unable to find the specified registry key or value.
```

```
reg load HKLM\zCOMPONENTS $mainOSDrive\scratchdir\Windows\System32\config\COMPONENTS
reg load HKLM\zDEFAULT $mainOSDrive\scratchdir\Windows\System32\config\default
reg load HKLM\zNTUSER $mainOSDrive\scratchdir\Users\Default\ntuser.dat
reg load HKLM\zSOFTWARE $mainOSDrive\scratchdir\Windows\System32\config\SOFTWARE
reg load HKLM\zSYSTEM $mainOSDrive\scratchdir\Windows\System32\config\SYSTEM
```

**Comment by wazerstar** [comment]
Ok I decided to rename Administrators > Administratorer in the script file and reran, it was much better but still errors showing up..


[mod_tiny11.log](https://github.com/ntdevlabs/tiny11builder/files/15181884/mod_tiny11.log)

**Comment by ntdevlabs** [comment]
Yeah, looks like almost all of the issues are due to the Administrator user group having a different name in Danish. However, it's odd that it can't fully unmount the image. Have you tried restarting the computer?

**Comment by wazerstar** [comment]
> Yeah, looks like almost all of the issues are due to the Administrator user group having a different name in Danish. However, it's odd that it can't fully unmount the image. Have you tried restarting the computer?

Yeah I did and still same, I even did use Remove_Failure_MountDir_TempDir.cmd
[Remove_Failure_MountDir_TempDir.txt](https://github.com/ntdevlabs/tiny11builder/files/15181954/Remove_Failure_MountDir_TempDir.txt)

Which successful unmounted the dirs, but running the script after it still said it again.

I guess you can use this to detect different administrator languages and then replace "Administrators" with `$AdminGroupName`


`$AdminGroupName = (Get-WmiObject -Class Win32_Group -Filter 'LocalAccount = True AND SID = "S-1-5-32-544"').Name`

**Comment by ntdevlabs** [comment]
Already working on a fix. Now I'm testing it
![image](https://github.com/ntdevlabs/tiny11builder/assets/38796849/969f42de-b08e-4c20-98a5-7aaffea45a36)

**Comment by ntdevlabs** [comment]
I just pushed a new update that should fix this issue. Let me know how it goes!

**Comment by wazerstar** [comment]
> I just pushed a new update that should fix this issue. Let me know how it goes!

You are still using Administrators in other parts of the script which needed to be replaced as well and then still other errors.

https://i.imgur.com/ozZK9DN.png

[tiny11.log](https://github.com/ntdevlabs/tiny11builder/files/15182078/tiny11.log)

tiny11.iso is around 5.7gigs not tiny :)

**Comment by ntdevlabs** [comment]
Yeah, you're right. Haven't noticed those, sorry about that. Testing the changes again :)
